bigframes/session/loader.py

```diff
@@ -449,6 +450,20 @@ def read_gbq_table(
         # if we don't have a unique index, we order by row hash if we are in strict mode
         if self._force_total_order:
             if not primary_key:
+                # 5gb chosen rather arbitrarily, mostly want to avoid warning for very small tables, for which the
+                # price is trivial
+                if (table.num_bytes or 0) > 5000000000:
```

Comment:
Nit: Might be good to do a constant here or at least use _ as a 0 separator.

```suggestion
                if (table.num_bytes or 0) > 5_000_000_000:
```